pkgs/dart_tooling_mcp_server/test_fixtures/counter_app/analysis_options.yaml

```diff
@@ -0,0 +1,28 @@
+# This file configures the analyzer, which statically analyzes Dart code to
+# check for errors, warnings, and lints.
+#
+# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
+# IDEs (https://dart.dev/tools#ides-and-editors). The analyzer can also be
+# invoked from the command line by running `flutter analyze`.
+
+# The following line activates a set of recommended lints for Flutter apps,
+# packages, and plugins designed to encourage good coding practices.
```

Comment:
nit: consider removing comments in this file to make it shorter

Reply:
This is all just the default sample app, I don't really want to touch it if that is OK (we may want to update it in the future and it should be as simple as just `flutter create`).

Reply:
Changed this to just import the dart_flutter_team_lints